tests/null_fk_ordering/models.py

```diff
@@ -2,7 +2,7 @@
 Regression tests for proper working of ForeignKey(null=True). Tests these bugs:
 
     * #7512: including a nullable foreign key reference in Meta ordering has un
-xpected results
+expected results
```

Comment:
Yeah, I realized it was related (I had to look in order to figure out what the word was).
I can pull `un` onto the second line.